### PR TITLE
4 - further changes to nifti nibabel

### PR DESCRIPTION
**Pull request by mxochicale**
# Addressing #4

### CODE REVIEW

**Review by thompson318 — COMMENTED** [review]
Looking good. Probably best to wait for some feedback from Dave or Jamie before merging.

**Comment by jamie-mcclelland** [comment]
thanks @mxochicale for this - I'm afraid I don't have time to give any detailed feedback at the moment, but from the quick look I just had at all the changes it looks all good in general.

**Review by zsenousy — APPROVED** [review]
Looks good. Happy to approve.

File: episodes/excthreesteps.Rmd
Comment (davecash75) [inline episodes/excthreesteps.Rmd:41]: I'm very cautious about using PYTHONPATH in conjunction with conda, as that PYTHONPATH will stay the same regardless of what environment you are using, so any modules in PYTHONPATH could end up clashing with different versions and modules that you have loaded in various environments. I'm not sure whether we should ahve this in the text

**Comment by davecash75** [comment]
@mxochicale @thompson318 @jamie-mcclelland  I'm not quite sure how this branch addresses the issue which Jamie initially asked for which is that we wanted to put in more detail around Nibabel on Exercise 4. This seems to be correcting a lot of various issues. Also, I think we need to open it up again (or I can just make another issue/branch) as there are a few typos and changes to sentences that we need to sort out.

**Comment by mxochicale** [comment]
@davecash75 I am afraid I cannot do much at this stage as I requested reviews from Thursday 27-June at 20:09, offering to add further changes over the weekend of 29 and 30 of June. So, anything from 1st of August can be leave for future work or assigned to yourself or @jamie-mcclelland 
I strongly suggest to opening new issues following from your comments and follow this [guidelines](https://github.com/HealthBioscienceIDEAS/Medical-Image-Registration-Short-Course/blob/main/CONTRIBUTING.md#pull-request-pr-and-merge-to-main-branch) to sort them out. We have also this milestone that pointed out clearly the end of our work "Deadline for delivery of sandpaper for Medical-Image-Registration-Short-Course is 30-June-2024": https://github.com/HealthBioscienceIDEAS/Medical-Image-Registration-Short-Course/milestone/1 
Hope that make sense, happy to chat when we meet! 

Miguel

**Pull request merged**
